bin/varnishtest/tests/d00007.vtc

```diff
@@ -53,10 +71,24 @@ varnish v1 -vcl {
 
 	backend dummy { .host = "${bad_backend}"; }
 
-	probe pr {
+	probe p1 {
+		.threshold = 8;
+		.initial = 8;
+		.interval = 1m;
+		.request = "GET / HTTP/1.1"
+		           "Host: ${s1_addr}"
+		           "Probe: p1"
+			   "Connection: close";
+	}
+
+	probe p2 {
 		.threshold = 8;
 		.initial = 8;
 		.interval = 1m;
+		.request = "GET / HTTP/1.1"
+		           "Host: ${s1_addr}"
+		           "Probe: p2"
+			   "Connection: close";
```

Comment:
Instead of space-aligning after `.request = ` I would break the line after the `=` sign and simply use one more regular tab indentation.